### ISSUE TITLE
[remoteopenhab][docker] Communication error

### ISSUE THREAD

**Issue by bwosborne2** [openhab/openhab-addons#9105]
## Expected Behavior
The Report openHAB Server Bridge should be Online

## Current Behavior
After initial issues after upgrading to snapshot build 2025, I deleted and re-added the report server. Here is an excerpt from the log

```
2020-11-22 18:22:59.372 [INFO ] [me.event.ThingStatusInfoChangedEvent] - Thing 'remoteopenhab:server:c5125ee170' changed from UNKNOWN to OFFLINE (COMMUNICATION_ERROR): Failed to execute the root REST API
```
This has been working fine through several snapshot updates. The OH2 server is accessible from its Web UI. The API is accessible from REST Docs too. FYI my OH2 server is on port 8081 but I set that in the Thing definition.

## Steps to Reproduce (for Bugs)

1. Install Snapshot 2025 & reportopenhab
2. Try to add a remote server bridge and observe the behaviour.

## Your Environment

* Docker openHAB 3.0.0-snapshot Build 2025
* Raspbian Lite on a Pi 3B+

**Comment by lolodomo** [comment]
Please enable the binding DEBUG logs, at least it will show what URL is used.

**Comment by lolodomo** [comment]
And please check what binding version you are running because I am expecting a more detailed error message with the current version. 
binding:list -s | grep remoteopenhab

**Comment by bwosborne2** [comment]
> binding:list -s | grep remoteopenhab
```
openhab> binding:list -s | grep remoteopenhab
gogo: CommandNotFoundException: Command not found: binding:list
```
How about this? ;)
```
openhab> bundle:list -s | grep org.openhab.binding.remoteopenhab
208 x Active x  80 x 3.0.0.202011092021      x org.openhab.binding.remoteopenhab
```

I will try and enable DEBUG logs later.

**Comment by lolodomo** [comment]
First, as I supposed, you are running an old version built 14 days ago.

**Comment by bwosborne2** [comment]
OK is the Milestone newer then? If so I can try that later.

**Comment by lolodomo** [comment]
Of course, there is a more recent version in snapshots and in Milestone 3.
With my snapshot 2024:
```
bundle:list -s | grep remote
225 x Active x  80 x 3.0.0.202011210347      x org.openhab.binding.remoteopenhab
```
Built the 21th of November.

**Comment by bwosborne2** [comment]
Apparently not all snapshots make it to Docker Hub. I just installed this shortly before opening the issue.

The development system that worked reasonably well for 2.5.0 has deteriorated rapidly. One reason I chose openHAB was because of its good development policy of nightly snapshots, milestones, and then stable releases.  Even then there was at least one ignored bug in 2.5.0 that broke things for new users that installed the addons package.

Sorry for the rant.
